gitlab/resource_gitlab_project_hook.go

```diff
@@ -90,6 +90,11 @@ func resourceGitlabProjectHook() *schema.Resource {
 				Optional: true,
 				Default:  false,
 			},
+			"releases_events": {
```

Comment:
Please add the `Description` field here instead of in the Markdown file and run `make reviewable`.